thunderfish/sorting_tools.py

```diff
@@ -29,16 +30,17 @@ def filter_fishes(fishlists):
     return fishlist
 
 
-def wave_or_pulse_psd(power, freqs, data, rate, fresolution, create_dataset=False):
-    if create_dataset is True:
-        fig, ax = plt.subplots()
-        plt.axis([0, 3000, -110, -30])
-        ax.plot(freqs, 10.0 * np.log10(power))
+def wave_or_pulse_psd(power, freqs, data, rate, fresolution, create_dataset=False, kategory='pulse'):
+    # if create_dataset is True:
+    #    fig, ax = plt.subplots()
+    #    plt.axis([0, 3000, -110, -30])
+    #    ax.plot(freqs, 10.0 * np.log10(power))
```

Comment:
again: ```category```
you are under version control, if the lines are not needed, remove them